### ISSUE TITLE
Unable to undo bad relabeling

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#2165]
I relabled a command with a c-shape argument to one with fewer arguments and it went away. Typing control-z made me see the lost code fly off very fast to the current palette.

It seems this is like #696 which was closed because undo was to be available but I just tested it and I couldn't undo what was lost when deleting an argument.

**Comment by cycomachead** [comment]
Yeah, this seems like a bug. The undo-of-undo is a problem, and can lead to losing code. :(
